### PR TITLE
Push coverage reports to codecov

### PR DESCRIPTION
**Pull request by WhyNotHugo**
Submit coverage report to codecov:

https://codecov.io/gh/sgrepo/celery-unique/branch/coverage

Also added CI/coverage badges to the README.

### CODE REVIEW

**Comment by WhyNotHugo** [comment]
Ugh, this damn useless bot. I think it can only be disabled at an org level.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
 # celery-unique
+
+![Travis CI build status](https://img.shields.io/travis/sgrepo/celery-unique.svg)
+![Codecov coverage
+report](https://img.shields.io/codecov/c/github/sgrepo/celery-unique.svg)
```
Comment (TylerHendrickson) [inline README.md]: Errant line break here; should be
```markdown
![Codecov coverage report](https://img.shields.io/codecov/c/github/sgrepo/celery-unique.svg)
```

**Review by TylerHendrickson — CHANGES_REQUESTED** [review]
@hobarrera 1 small change

**Review by TylerHendrickson — APPROVED** [review]
@hobarrera Looks good!

**Pull request merged**